torch/_inductor/constant_folding.py

```diff
@@ -220,6 +220,11 @@ def set_env(arg: torch.fx.Node) -> None:
         ):
             return self.unknown_value
 
+        if node.op == "call_function" and isinstance(
+            node.target, torch._higher_order_ops.torchbind.CallTorchBind
```

Comment:
I don't think we wanna specialize on call_torchbind here, in fact, we probably should exclude all HOOs here to be safe.